x-pack/plugins/apm/server/routes/errors/get_error_groups/get_error_group_summary.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { rangeQuery, kqlQuery } from '@kbn/observability-plugin/server';
+import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import { asMutableArray } from '../../../../common/utils/as_mutable_array';
+import {
+  ERROR_CULPRIT,
+  ERROR_EXC_HANDLED,
+  ERROR_EXC_MESSAGE,
+  ERROR_GROUP_ID,
+  ERROR_LOG_MESSAGE,
+  SERVICE_NAME,
+  TRANSACTION_SAMPLED,
+} from '../../../../common/es_fields/apm';
+import { environmentQuery } from '../../../../common/utils/environment_query';
+import { APMEventClient } from '../../../lib/helpers/create_es_client/create_apm_event_client';
+
+export async function getErrorGroupSummary({
+  environment,
+  kuery,
+  serviceName,
+  groupId,
+  apmEventClient,
+  start,
+  end,
+}: {
+  environment: string;
+  kuery: string;
+  serviceName: string;
+  groupId: string;
+  apmEventClient: APMEventClient;
+  start: number;
+  end: number;
+}) {
+  const params = {
+    apm: {
+      events: [ProcessorEvent.error as const],
+    },
+    body: {
+      track_total_hits: true,
+      size: 0,
+      query: {
+        bool: {
+          filter: [
+            { term: { [SERVICE_NAME]: serviceName } },
+            { term: { [ERROR_GROUP_ID]: groupId } },
+            ...rangeQuery(start, end),
+            ...environmentQuery(environment),
+            ...kqlQuery(kuery),
+          ],
+          should: [{ term: { [TRANSACTION_SAMPLED]: true } }],
+        },
+      },
+      sort: asMutableArray([
+        { _score: { order: 'desc' } }, // sort by _score first to ensure that errors with transaction.sampled:true ends up on top
+        { '@timestamp': { order: 'desc' } }, // sort by timestamp to get the most recent error
+      ] as const),
+    },
+    aggs: {
+      grouping_key: {
+        terms: {
+          field: 'error.grouping_key',
+          size: 1,
+        },
+        aggs: {
+          error: {
+            top_hits: {
+              _source: {
+                includes: [
+                  ERROR_EXC_MESSAGE,
+                  ERROR_EXC_HANDLED,
+                  ERROR_LOG_MESSAGE,
+                  ERROR_CULPRIT,
+                ],
+              },
+            },
+          },
+        },
+      },
```

Comment:
We don't need an aggregation here. Instead set `size: 1` and specify the fields in `_source`.

Reply:
I think for the fields listed here there will only be a single value for a specific error group id. But you might want to double check with APM Server.